engines/libaio.c

```diff
@@ -228,6 +238,20 @@ static int fio_libaio_getevents(struct thread_data *td, unsigned int min,
 		} else {
 			r = io_getevents(ld->aio_ctx, actual_min,
 				max, ld->aio_events + events, lt);
+			if (td->o.zone_mode == ZONE_MODE_ZBD
+			    || td->o.zone_mode == ZONE_MODE_STRIDED) {
```

Comment:
Same comment as above. Why consider strided mode here ?

Reply:
Strided mode support will be removed in the v2 patches.